ui-v2/app/components/service-list/index.hbs

```diff
@@ -0,0 +1,17 @@
+{{yield}}
+{{#if (gt items.length 0)}}
+<ListCollection @cellHeight={{73}} @items={{items}} @class="service-list" as |item index|>
```

Comment:
Is there a way we can pass this through with `class=""` (so without the `@`) and then use `...arguments` inside the component? I'm not sure if its possible or not with `ember-collection` being quite old now, but maybe we could give it a go. Shout me if you don't follow what I mean here.

Reply:
I don't follow. Is there a template file using `...arguments` that I could follow?

Reply:
Oh sorry its `...attributes` not `...arguments` 😆 

Here's one:

https://github.com/hashicorp/consul/blob/ui-staging/ui-v2/app/components/toggle-button/index.hbs

Reply:
Passing `...attributes` down to EmberNativeScrollable breaks.

Reply:
Ok I think what we need to do here is remove the `@class` from `EmberNativeScrollable` and change `@class="service-list"` to just `class="service-list"` on `ListCollection` inside `<ConsulServiceList />`.

If we can get away with using DOM `class` as opposed to `@class` then great. There's no `...attributes` needed I don't think ('cause ListCollection isn't yet a tagless/outerHTML component)

We can catch up on this tomorrow though, don't worry too much, I think the main thing is to take a look at that responsiveness.

Reply:
Removing the `@` in both places works.